packages/lumx-core/src/scss/components/button/_mixins.scss

```diff
@@ -130,7 +130,7 @@
         }
     }
 
-    &:hover,
+    &:hover:enabled,
```

Comment:
When using the Button component with an `href` (instead of `onClick`) then we use `<a>` instead of `<button>` and it looks like it does not work with the `:enabled` CSS pseudo class. 
In the story "With Href", I don't see the hover state even though the Button is not disabled :S